core/locale/domain/dateTime.go

```diff
@@ -17,6 +23,29 @@ func (dtf *DateTimeFormatter) SetDateTime(time time.Time, localtime time.Time) {
 	dtf.localDateTime = localtime
 }
 
+// SetLocation sets location for local time
+func (dtf *DateTimeFormatter) SetLocation(loc string) *DateTimeFormatter {
+	location, err := time.LoadLocation(loc)
+
+	if err != nil {
+		if dtf.logger != nil {
+			dtf.logger.WithField(flamingo.LogKeyMethod, "SetLocation").Error(
+				fmt.Sprintf("%s: failed to load time zone location: %v", loc, err),
+			)
+		}
+	} else {
+		dtf.localDateTime = dtf.dateTime.In(location)
+	}
+
+	return dtf
+}
+
+// SetLogger sets the logger instance
+func (dtf *DateTimeFormatter) SetLogger(logger flamingo.Logger) {
```

Comment:
With my comments above we won't need a logger in here.

Reply:
I agree. I think logger should not be a part of DateTimeFormatter. Method "SetLocation" already returns an error, if there is one, so it can be logged from the place when the method is called. :)